.eslintrc.json

```diff
@@ -39,6 +39,7 @@
         "@typescript-eslint/no-use-before-define": 0,
         "class-methods-use-this": 0,
         "no-useless-constructor": 0,
+        "quotes": ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": false }],
```

Comment:
template strings are magical, so it's best to only use them when necessary. as it happens, `allowTemplateLiterals` being false only forbids template literals when there are no variables inside